lib/teleterm/clusters/cluster_auth_test.go

```diff
@@ -68,9 +71,10 @@ func TestPwdlessLoginPrompt_PromptTouch(t *testing.T) {
 		return nil
 	}
 
-	prompt := newPwdlessLoginPrompt(context.Background(), stream)
-	err := prompt.PromptTouch()
+	prompt := newPwdlessLoginPrompt(context.Background(), log, stream)
+	acknowledgeTouch, err := prompt.PromptTouch()
```

Comment:
```suggestion
	ackTouch, err := prompt.PromptTouch()
```

Might as well, since I tagged most of the others too.